test/VT_DATE/getvardate.js

```diff
@@ -67,7 +67,15 @@ vdate = new Date(1234567890123).getVarDate();
 safeCall(function() { writeLine(vdate ? true : false); });
 safeCall(function() { writeLine([1, 2].indexOf(2, vdate)); }); // valid only in version 3
 safeCall(function() { writeLine(parseInt("1", vdate)); });
-safeCall(function() { writeLine([1, vdate, 2].toLocaleString()); });
+
+// normalize Old GetNumberFormatEx exception
+safeCall(function() {
+    var result = [1, vdate, 2].toLocaleString();
+    if (result == "1, [object Object], 2") {

```

Comment:
Interesting- do we need to fix the Windows case or are you leaving it here for compat? @abchatra- thoughts?

---
In reply to: [65751957](https://github.com/Microsoft/ChakraCore/pull/1065#discussion_r65751957) [](ancestors = 65751957)